.github/actions/release/bump-versions/action.yml

```
@@ -0,0 +1,90 @@
+name: 'bump-versions'
+description: 'Bump versions in files for release target'
+inputs:
+  release-target:
+    description: "target of the release (rust|wasm)"
+    required: true
+  version:
+    description: "release name"
+    required: true
+
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        profile: minimal
+        override: true
+
+    - name: Install cargo-release
+      uses: actions-rs/cargo@v1
+      with:
+        command: install
+        args: cargo-release
+
+    - name: Install cargo-edit # to use cargo add
+      uses: actions-rs/cargo@v1
+      if: ${{inputs.release-target == 'rust'}}
+      with:
+        command: install
+        args: cargo-edit
```

We can install just the "add" command, which might be slightly faster. We should also pin it to a particular version.

```suggestion
    - name: Install cargo-edit # to use cargo add
      uses: actions-rs/cargo@v1
      if: ${{inputs.release-target == 'rust'}}
      with:
        command: install
        args: -f --no-default-features --features "add" --version ^0.8 cargo-edit
```